yarn.lock

```diff
@@ -16526,6 +16533,11 @@ unbox-primitive@^1.0.1:
     has-symbols "^1.0.2"
     which-boxed-primitive "^1.0.2"
 
+undici@^4.14.1:
```

Comment:
I am always worried about additional dependencies, but good choice that it does not have more dependencies! :)

Reply:
It won't actually be bundled with Alby and only used in Node.js environments. Once Node 18 is out, we'll soon drop the undici dependency completely because node will have the fetch API natively.